src/test/java/pages/BasePage.java

```diff
@@ -2,7 +2,7 @@
 
 import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.SelenideElement;
-import org.openqa.selenium.By;
+
 
 import java.time.Duration;
 
```

Comment:
на строке 19 -20 ненужный sout

Reply:
14 - 15 строки, странное название, может имелось ввиду "navigationToCart"?
Локатор используется один и тот же, можно сделать так: 
```suggestion

    private final SelenideElement notificationToCart = $x(".//p[@class='content']");
    private final SelenideElement notificationToCartBtn = notificationToCart.$x("./descendant::a[@href='/cart']");
```

точки нужно добавлять вначале локаторов, чтобы иметь возможность создавать один на основе другого